### DISCUSSION TITLE
old visual studio project cannot recognize vcpkg packages, but newly created ones can

### DISCUSSION THREAD

**Discussion by xarthurx** [microsoft/vcpkg#24940] [Q&A]
I got a new machine and installed `vcpkg` and `visual studio 2022`.

New cpp project created inside Visual Studio can recognize and `#include` packages installed by vcpkg.
However, I pulled my old project from github and compile them, vcpkg-installed packages are not recognized...

I'm sure `triplet` is set correctly. All `x64-windows`.

Any ideas?

**Comment by xarthurx** [comment]
OK, somehow I delete the whole vcpkg and re-installed it, it works.

**Reply by BaikenM** [reply]
Did you have to delete and reinstall all the libraries?

**Reply by xarthurx** [reply]
Yes. For my case, I did it.

> Did you have to delete and reinstall all the libraries?

**Comment by BaikenM** [comment]
You could also set up a new project and transfer files from the old project there, instead of reinstalling vcpkg and all the dependencies every time you try to build an old project, but it's obviously suboptimal. It must boil down to project properties, I fiddled around with them with no luck so far.
